cplusplus/include/vips/VImage8.h

```diff
@@ -2994,7 +2994,7 @@ class VImage : public VObject {
 	 * **Optional parameters**
 	 *   - **n** -- Number of bands to extract, int.
 	 *
-	 * @param band Band to extract.
+	 * @param band Index of a first band to extract.
```

Comment:
This file is generated automatically, so you don't need to change this yourself.